public/docs/ts/latest/guide/template-syntax.jade

```diff
@@ -799,22 +867,31 @@ a#event-binding
   keystrokes, mouse movements, clicks, and touches.
   You declare your interest in user actions through Angular event binding.
 
-  Event binding syntax consists of a **target event** within parentheses on the left of an equal sign, and a quoted
+  Event binding syntax consists of a **target event** name
+  within parentheses on the left of an equal sign, and a quoted
   [template statement](#template-statements) on the right.
-  The following event binding listens for the button's click event, calling
+  The following event binding listens for the button's click events, calling
```

Comment:
Why plural "events"?  I'm talking about the "click event", not the stream of events passed to the click event handler.  May I should have written 
>for the button's `click` event